content/doc/book/managing/nodes.adoc

```diff
@@ -90,9 +90,11 @@ We take advantage of the Windows Scheduler's ability to run command at system st
 
 [source,java]
 ----
-java -jar agent.jar \
--jnlpUrl http://<JenkinsHostName>:8080/computer/<nodeName>/slave-agent.jnlp \
--secret <some_long_hex_string>
+java \
```

Comment:
Here and elsewhere in this file, migrate from deprecated `-jnlpUrl` to non-deprecated equivalent.